config/application.rb

```diff
@@ -2,11 +2,16 @@
 
 require 'rails/all'
 
+# TODO: test if this is (still) required
```

Comment:
@naseberry unrelated but noticed this and not sure if we are using [susy]( https://www.oddbird.net/susy/)

Reply:
@jsugarman  Susy is being used by our style sheet, however it is now loaded via npm.

I need to create a task to test and if required remove redundant rails dependencies and configuration related to asset management.

Reply:
cool, can we remove the comment then

Reply:
@jsugarman yes please